src/generated/xproto.rs

```diff
@@ -1044,6 +1045,7 @@ impl Serialize for Setup {
 }
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[allow(non_camel_case_types)]
```

Comment:
Huh?

Reply:
Indeed, useless. I'll fix it.

Reply:
Well, mainly I am curious why this was not necessary before and is necessary now. One `git grep` later: Oh, the current code generator already emits this in some cases. So, I guess this comment can just be ignored. I'll mark the conversation as resolved.

However, `DriverType`, `Option`, `Capability`, `AtomEnum`, and `ImageFormatInfoType` all look quite camel-case-y to me, so the Python code generator might already emit this `#[allow]` too much...? Dunno if `GC` and `RM` count as camel case.
Well, whatever...

Reply:
The lint is there for the names of the variants, not for the name of the enum (e.g., `AtomEnum` has `CUT_BUFFER0`).

Reply:
Oh, obvious. Thanks & Sorry.